docs/guides/federation.md

```diff
@@ -138,50 +181,87 @@ the requested `id` and a fixed number of reviews.
 If we were to add more fields to `Book` that were stored in a database, this
 would be where we could perform queries for these fields' values.
 
-Now we need to do is to define a `Query` type, even if our service only has one
-type that is not used directly in any GraphQL query. This is because the GraphQL
-spec mandates that a GraphQL server defines a Query type, even if it ends up
-being empty/unused. Finally we also need to let Strawberry know about our Book
-and Review types. Since they are not reachable from the `Query` field itself,
-Strawberry won't be able to find them by default.
+We also defined a `Query` type that has a single field, `_hi`, which returns a
+string. This is required because the GraphQL spec mandates that a GraphQL server
+defines a Query type, even if it ends up being empty/unused.
 
-```python
-@strawberry.type
-class Query:
-    _service: Optional[str]
+Finally we also need to let Strawberry know about our Book and Review types.
+Since they are not reachable from the `Query` field itself, Strawberry won't be
+able to find them.
 
-schema = strawberry.federation.Schema(query=Query, types=[Book, Review])
+## Let's run our services
+
+Before starting Apollo Router to compose our schemas we need to run the
+services.
+
+In two terminal windows, run the following commands:
+
+```bash
+cd books
+strawberry server --port 3500 app
+```
+
+```bash
+cd reviews
+strawberry server --port 300 app
```

Comment:
I was reading through the docs preview (awesome work by the way with this PR -- and strawberry in general) and noticed this typo: `300` => `3000`.